.circleci/config.yml

```diff
@@ -1071,7 +1071,6 @@ jobs:
             ./Tests/scripts/slack_notifier.sh 'test_playbooks' ./env_results.json
           when: always
       - *comment_on_contrib_pr
-      - *destroy_instances
```

Comment:
?

Reply:
just for testing, she wants to log into the instance